### DISCUSSION TITLE
How to map the grain ID to the pixel raw data ID?

### DISCUSSION THREAD

**Discussion by euerka** [mtex-toolbox/mtex#1478] [Ask Anything] [answered]
As we knew that we can assign grain ID for all grains, and we know each grain contains how many pixels.
For example, Grain ID 40 includes 200 pixels, my question is may I know what those specific 200 pixels are?

In the raw pixel data, normally we know pixel ID and also the position of each pixel, how can we link them to grains?

Thanks!

Regards,
-cheng xi

**Comment by zmichels** [comment, marked as answer]
After computing grains, check the “grainId” property of the EBSD variable:
ebsd.grainId
